src/views/LimitedRestorationToFull.vue

```diff
@@ -297,13 +298,13 @@ export default class LimitedRestorationToFull extends Mixins(
 
       this.setEntitySnapshot(entitySnapshot)
 
-      // parse draft restoration filing into store
-      this.parseRestorationFiling(restorationFiling)
-
       // set the previously filed limited restoration in the store
       // (will throw on error)
       await this.setStateFilingRestoration()
 
+      // parse draft restoration filing into store
+      this.parseRestorationFiling(restorationFiling)
+
```

Comment:
THIS is the bug fix -- we need to set the previously file limited restoration in the store (line 303) before we parse the draft filing into the store (since the parse code needs the state filing).